client/app/components/TimeAgo.jsx

```diff
@@ -6,6 +6,25 @@ import PropTypes from 'prop-types';
 import { Moment } from '@/components/proptypes';
 import { clientConfig } from '@/services/auth';
 import useForceUpdate from '@/lib/hooks/useForceUpdate';
+import Tooltip from 'antd/lib/tooltip';
+
+moment.updateLocale('en', {
```

Comment:
As this is something that applies globally it probably should be somewhere else (`app/config`?).

Reply:
👍 56b6571cca7da6b39192413069615885be957b8b